users/serializers.py

```diff
@@ -40,7 +40,9 @@ class PrizeSerializer(serializers.Serializer):
     value = serializers.IntegerField()
 
     def create(self, validated_data):
-        return Prize.objects.create(**validated_data)
+        student_data = validated_data.pop('student')
+        student = Student.objects.get(pk=student_data['id'])
```

Comment:
I think the same as above applies here. If I recall correctly you can use id and don't need to retrieve the whole object.